packages/orchestration/src/exos/local-chain-facade.js

```diff
@@ -70,17 +72,14 @@ const prepareLocalChainFacadeKit = (
           return watch(this.state.localChainInfo);
         },
 
-        // FIXME parameterize on the remoteChainInfo to make()
-        // That used to work but got lost in the migration to Exo
-        /** @returns {Vow<PromiseToVow<OrchestrationAccount<ChainInfo>>>} */
+        /** @returns {Vow<LocalOrchestrationAccountKit['holder']>} */
         makeAccount() {
           const lcaP = E(localchain).makeAccount();
-          // TODO #9449 fix types
-          // @ts-expect-error Type 'Vow<Voidless>' is not assignable to type 'Vow<OrchestrationAccountI>'.
           return watch(
-            // TODO #9449 fix types
-            // @ts-expect-error Property 'getAddress' does not exist on type 'EMethods<Required<Guarded<{ getAddress()...
-            allVows([lcaP, E(lcaP).getAddress()]),
+            // XXX makeAccount returns a Promise for an exo but reserves being able to return a vow
+            // so we use heapVowE to shorten the promise path
+            // eslint-disable-next-line no-restricted-syntax -- will run in one turn
+            allVows([lcaP, heapVowE(lcaP).getAddress()]),
```

Comment:
I tried to explain this in the comment. The value actually being returned is not a vow, but the return signature says it could be. @michaelfig wanted to reserve that option for the future. So this code should comply. (i.e. not to accelerate Hyrum's Law).

Reply:
That makes sense, thanks